archive/events/training/ionic-superpowers-tour.html

```diff
@@ -8127,10 +8127,10 @@ <h4>About the presenters</h4>
                           <p>
                             <img
                               class="thumbnail"
-                              src="https://sharepoint.ssw.com.au/AboutUs/Employees/PublishingImages/adamS_info.jpg"
+                              src="https://raw.githubusercontent.com/SSWConsulting/SSW.People.Profiles/main/Adam-Cogan/Images/Adam-Cogan-Profile.jpg"
```

Comment:
Guessing this is tech debt because of the missing CORS headers on requests to Rules

```suggestion
TODO: update after CORS issues are resolved for SSW Rules https://github.com/SSWConsulting/SSW.Cloudflare/issues/44
                              src="https://raw.githubusercontent.com/SSWConsulting/SSW.People.Profiles/main/Adam-Cogan/Images/Adam-Cogan-Profile.jpg"
```